lib/nodes/addon/components/driver-azure/template.hbs

```diff
@@ -264,13 +376,35 @@
         <label class="acc-label">
           {{t "nodeDriver.azure.size.label"}}
         </label>
-        {{new-select
-          classNames="form-control"
-          content=sizeChoices
-          optionLabelPath="value"
-          optionGroupPath="group"
-          value=config.size
-        }}
+        {{#if vmSizesLoading}}
+          <i class="icon icon-spinner icon-spin"></i>
+        {{else}}
+          {{new-select
+            classNames="form-control"
+            content=sizeChoices
+            optionLabelPath="value"
+            optionGroupPath="group"
+            value=config.size
+          }}
+          {{#if showVmSizeAvailabilityWarning}}
+            {{#banner-message color="bg-error"}}
+              <p>{{vmSizeAvailabilityWarning}}</p>
+            {{/banner-message}}
+          {{/if}}
+          {{#if showVmAvailabilityZoneWarning}}
```

Comment:
I'm not sure it always makes sense to show `vmSizeAvailabilityWarning` here, since one of the two potential error messages is asking the user to re-configure one of two options in a different section. `regionSupportsAzsButNotThisSize` certainly makes sense here, but probably not `regionDoesNotSupportAzs` :

![Screen Shot 2022-12-19 at 2 06 00 PM](https://user-images.githubusercontent.com/42977925/208524421-e2871d32-d9c4-491e-b888-a0d399d6b0e0.png)

Reply:
removed the vmSizeAvailabilityWarning

Reply:
The one that needs to go is actually `vmAvailabilityZoneWarning` completely my fault, sorry. Other than that the PR looks good